src/Containers/MiddleCanvas/DroppableContainerContents.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { Mixins } from '../../Utils';
+
+export interface DroppableContainerContentsProps extends MainInterface, ResponsiveInterface, React.HTMLAttributes<HTMLDivElement> {
+    styling: string,
```

Comment:
remove this entirely, and just style through te props spread

Reply:
This is the comment I meant. How would I know when to change it from space-between to centre (which is what I need for the different displays) without an indicator to do so?